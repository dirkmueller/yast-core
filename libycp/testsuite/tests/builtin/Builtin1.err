Parsed:
----------------------------------------------------------------------
{ return (1 + 2); }
----------------------------------------------------------------------
[Scanner] tests/builtin/Builtin1.ycp:21 Error: Undeclared identifier 'name'
Parsed:
----------------------------------------------------------------------
YError
----------------------------------------------------------------------
[Interpreter] tests/builtin/Builtin1.ycp:21 undeclared identifier
Parsed:
----------------------------------------------------------------------
{ return `name; }
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/builtin/Builtin1.ycp"
    return (3 + 4);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/builtin/Builtin1.ycp"
    return 7;
}
----------------------------------------------------------------------
