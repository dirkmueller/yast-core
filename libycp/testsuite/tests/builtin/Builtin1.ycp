
# ---------------------------------------------------------
#
#  Filename:    Builtin1.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------

# sleep() not tested


// The double backquotes `` are called "deep quote". They are used
// to prevent the enclosed expression from evaluation.

( ``( 1+2 ))
( ``(name) )	// error
( ``(`name) )
( ``{ return 3 + 4; } )
( { return 7; } )
