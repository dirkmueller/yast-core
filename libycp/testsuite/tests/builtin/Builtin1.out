((1 + 2))
(nil)
(`name)
(7)
(7)
